### ISSUE TITLE
Dpkg install error on Ubuntu 20.04

### ISSUE THREAD

**Issue by undaunt** [trapexit/mergerfs#771]
General description
===================
When attempting to install mergerfs on 20.04 with a `sudo dpkg -i` command, I receive an error as noted below.

Expected behavior
=================
The executable would be copied to /usr/local/bin just like on 18.04.

Actual behavior
===============
This is the received output:

```
sudo dpkg -i ../mergerfs_2.29.0-31-g4160a8e~ubuntu-focal_amd64.deb 
(Reading database ... 76892 files and directories currently installed.)
Preparing to unpack .../mergerfs_2.29.0-31-g4160a8e~ubuntu-focal_amd64.deb ...
Unpacking mergerfs (2.29.0-31-g4160a8e~ubuntu-focal) ...
dpkg: error processing archive ../mergerfs_2.29.0-31-g4160a8e~ubuntu-focal_amd64.deb (--install):
 unable to open '/usr/bin/mergerfs.dpkg-new': No such file or directory
Errors were encountered while processing:
 ../mergerfs_2.29.0-31-g4160a8e~ubuntu-focal_amd64.deb
```

**Precise** steps to reproduce the behavior
===========================================

On a vanilla Ubuntu 20.04 ISO server install, I did the following:

```
sudo apt update && sudo apt full-upgrade -y
sudo dpkg-reconfigure tzdata
sudo apt install fakeroot
```

Then proceeded with the directions on the README:

```
git clone https://github.com/trapexit/mergerfs.git
cd mergerfs
sudo tools/install-build-pkgs
make deb
sudo dpkg -i ../mergerfs_version_arch.deb
```

System information
==================
Please provide as much of the following information as possible:

* [N/A] mergerfs version: `mergerfs -V`
* [N/A] mergerfs settings: `cat /etc/fstab` or the command line arguments
* [5.4.0-39-generic #43-Ubuntu SMP Fri Jun 19 10:28:31 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux] Linux version: `uname -a`
* [dpkg - version 1.19.7 ] Versions of any additional software being used
* [N/A] List of drives, filesystems, & sizes: `df -h`

**Comment by seuffert** [comment]
i am having the same issue (ubuntu 19.10)

```
sudo apt install ../mergerfs_2.29.0-35-g8948fc2~ubuntu-eoan_amd64.deb
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'mergerfs' instead of '../mergerfs_2.29.0-35-g8948fc2~ubuntu-eoan_amd64.deb'
The following packages will be upgraded:
  mergerfs
1 upgraded, 0 newly installed, 0 to remove and 17 not upgraded.
Need to get 0 B/199 kB of archives.
After this operation, 541 kB of additional disk space will be used.
Get:1 /home/seuffert/dev/mergerfs_2.29.0-35-g8948fc2~ubuntu-eoan_amd64.deb mergerfs amd64 2.29.0-35-g8948fc2~ubuntu-eoan [199 kB]
(Reading database ... 188556 files and directories currently installed.)
Preparing to unpack .../mergerfs_2.29.0-35-g8948fc2~ubuntu-eoan_amd64.deb ...
Unpacking mergerfs (2.29.0-35-g8948fc2~ubuntu-eoan) over (2.28.3-27-g2f769e7~ubuntu-eoan) ...
dpkg: error processing archive /home/seuffert/dev/mergerfs_2.29.0-35-g8948fc2~ubuntu-eoan_amd64.deb (--unpack):
 unable to open '/usr/bin/mergerfs.dpkg-new': No such file or directory
Errors were encountered while processing:
 /home/seuffert/dev/mergerfs_2.29.0-35-g8948fc2~ubuntu-eoan_amd64.deb
N: Download is performed unsandboxed as root as file '/home/seuffert/dev/mergerfs_2.29.0-35-g8948fc2~ubuntu-eoan_amd64.deb' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by trapexit** [comment]
Should be fixed with #776.
